packages/@okta/vuepress-site/docs/guides/sign-into-web-app/create-okta-application/index.md

```diff
@@ -13,13 +13,11 @@ Before you can sign a user in, you need to create an Okta app integration that r
     > **Note:** It is important to choose the appropriate application type for apps that are public clients. Failing to do so may result in Okta API endpoints attempting to verify an app's client secret, which public clients aren't designed to have, hence breaking the sign-in or sign-out flow.
 1. Enter a name for your app integration (or leave the default value).
 1. Enter values for the **Login redirect URI**. This is the callback described in <GuideLink link="../define-callback">Understand the callback route</GuideLink>. Add values for local development (for example, `http://localhost:8080/authorization-code/callback`) and production (for example, `https://app.example.com/authorization-code/callback`).
-
+1. Include a URI in the **Initiate login URI** box to have Okta initiate the sign-in flow by redirecting to this endpoint (for example, `https://example:0000.com/login`). This triggers the client to send an authorize request. This URI is also used when users reset their passwords and need to be redirected back to your app.
 <!-- removed step 3 until the base uri is returned to the app wizard ui in feb 2021 -->
-<!--
-3. Add the **Base URI** of your application during local development, such as `http://localhost:3000`. Also, add any base URIs where your application runs in production, such as `https://app.example.com`.
+<!-- 3. Add the **Base URI** of your application during local development, such as `http://localhost:3000`. Also, add any base URIs where your application runs in production, such as `https://app.example.com`.
 -->
-
-7. Click **Save** to finish creating the Okta app integration.
+1. Click **Save** to finish creating the Okta app integration.
```

Comment:
Just curious, what is `1.` and `7.`? It seems they are not sequence number.

Reply:
there's a line break above step 7 - so it breaks the auto-numbering that markdown does. But, after 7, I can go back to 1., 1., 1., and so on.